.github/workflows/ci.yaml

```
@@ -0,0 +1,97 @@
+        id: cache-kwil-build
+        uses: actions/cache@v4
+        with:
+          path: tmp-node/.build
+          key: ${{ runner.os }}-kwil-build-${{ steps.node-commit.outputs.sha }}
+
+      - name: Build kwil-cli binary
+        if: steps.cache-kwil-build.outputs.cache-hit != 'true'
+        run: |
+          cd tmp-node
+          task build
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)